### ISSUE TITLE
[dx] Make it easy to get set up for development

### ISSUE THREAD

**Issue by mattwynne** [SmartBear/git-en-boite#19]
When @karamosky wanted to run the tests locally, he wanted to have redis and postgres running in Docker, but use is native machine for the app.

We struggled with:

- setting environment variables (there are lots of them)
- running just those docker containers, and exposing the ports

Possible solutions:
- better docs
- something like https://www.npmjs.com/package/dotenv for everyone to make it easy to set environment variables
- Something like https://docs.docker.com/compose/extends/ to have different docker compose contexts for these kinds of setups.

**Comment by mattwynne** [comment]
@jbpros, @blethier and I did somework on this today. We swapped out learna for yarn workspaces, and tidied up the README with accurate setup instructions.

We still need to solve the problem of getting people a quick dev environment when they don't want to install redis / postgres locally.

It looks like docker-compose's extends functionality is [gone in the latest version](https://github.com/moby/moby/issues/31101), so probably that's a dead-end.
